openstudio-extension.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'openstudio-workflow', '~> 1.3.4'
   spec.add_dependency 'openstudio_measure_tester', '~> 0.1.7'
   spec.add_dependency 'parallel', '~> 1.12.0'
+  spec.add_dependency "github_api", "~> 0.18.0"
```

Comment:
@vtnate  -- I'm concerned about adding the gem dependency. As far as I can remember OpenStudio will need to be able to load this Gem (and dependent gems). I need to verify this with @kbenne

Reply:
I hear ya. There are definitely some things I don't yet understand about Ruby packages. As I mentioned above, OpenStudio-server doesn't have a `gemspec` file, but its `change_log.rb` file requires `github_api`. What am I missing?